src/features/Dashboard/InstructorAssignSection/ClassCard.jsx

```diff
@@ -19,8 +20,30 @@ const ClassCard = ({ data }) => {
 
   return (
     <div className="class-card-container">
-      <h4>{data?.className}</h4>
-      <p className="course-name">{data?.masterCourseName}</p>
+      <OverlayTrigger
```

Comment:
The tooltip is show also in the case when the name is not truncated, that is the behavior expected?

Reply:
@AuraAlba addressed!